src/raftstore/store/debug.rs

```diff
@@ -151,6 +166,214 @@ impl Debugger {
             Err(e) => Err(box_err!(e)),
         }
     }
+
+    pub fn scan_mvcc<D, E>(&self, mut req: ScanMvccRequest, deal: &mut D) -> Result<()>
```

Comment:
This function is too complicated and hard to read. Why not return an iterator instead?

Reply:
Got it. I will try it later.